Source: Gentoo, https://gitweb.gentoo.org/repo/gentoo.git/tree/net-libs/nodejs/files/nodejs-99999999-llhttp.patch
Upstream: llhttp will be the new default replacing http_parser, https://github.com/nodejs/node/commit/2cb8f247512d611d3dfe81c2441e8028bfb0e8f0
Reason: Fix build

--- a/node.gypi
+++ b/node.gypi
@@ -169,11 +169,15 @@
 
     [ 'node_shared_http_parser=="false"', {
       'dependencies': [
-        'deps/http_parser/http_parser.gyp:http_parser',
-        'deps/llhttp/llhttp.gyp:llhttp'
+        'deps/http_parser/http_parser.gyp:http_parser'
       ],
     } ],
 
+    [ '"true"', { 'dependencies': [
+        'deps/llhttp/llhttp.gyp:llhttp'
+      ],
+    }],
+
     [ 'node_shared_cares=="false"', {
       'dependencies': [ 'deps/cares/cares.gyp:cares' ],
     }],
